src/utils/decrypt-data.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  EncryptionContext,
+  KmsKeyringNode,
+  buildDecrypt,
+} from "@aws-crypto/client-node";
+import buildKmsKeyring from "./kms-keyring-builder";
+import { logger } from "./logger";
+import { getHashedAccessCheckValue } from "./get-access-check-value";
+
+const MAX_ENCRYPTED_DATA_KEY = 5;
+const DECODING = "utf8";
+const ENCODING = "base64";
+
+const decryptClientConfig = { maxEncryptedDataKeys: MAX_ENCRYPTED_DATA_KEY };
+const decryptClient = buildDecrypt(decryptClientConfig);
+
+let keyring: KmsKeyringNode;
+
+export async function generateExpectedContext(
+  userId: string
+): Promise<EncryptionContext> {
+  const { AWS_REGION, ACCOUNT_ID, ENVIRONMENT, VERIFY_ACCESS_VALUE } =
+    process.env;
+  if (AWS_REGION === undefined) {
```

Comment:
Consider using node assert in a try catch to improve readability of these guard conditions.

Reply:
Done